boards/nordic/nrf9280pdk/nrf9280pdk_nrf9280_cpurad.yaml

```
@@ -0,0 +1,19 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+identifier: nrf9280pdk/nrf9280/cpurad
+name: nRF9280-DK-nRF9280-Radio
+type: mcu
+arch: arm
+toolchain:
+  - gnuarmemb
+  - xtools
+  - zephyr
+sysbuild: true
+ram: 32
+flash: 368
```

This should match the size of `cpurad_slot0_partition`
```suggestion
flash: 256
```

---

Updated